www/main/navigator/search/components/SearchContent.jsx

```diff
@@ -151,7 +165,31 @@ const SearchContent = () => {
         <GurmukhiKeyboard searchType={currentSearchType} query={query} setQuery={setQuery} />
       )}
       <div className="search-result-controls">
-        <span>{filteredShabads.length ? `${filteredShabads.length} Results` : ''}</span>
+        {(currentWriter !== 'ALL' || currentRaag !== 'ALL' || currentSource !== 'all') && (
+          <div className="filter-tag--container">
+            {currentWriter !== 'ALL' && (
+              <FilterTag
+                close={() => setCurrentWriter('ALL')}
+                title={currentWriter}
+                filterType={i18n.t('SEARCH.WRITER')}
+              />
+            )}
+            {currentRaag !== 'ALL' && (
+              <FilterTag
+                close={() => setCurrentRaag('ALL')}
+                title={currentRaag}
+                filterType={i18n.t('SEARCH.RAAG')}
+              />
+            )}
+            {currentSource !== 'all' && (
+              <FilterTag
+                close={() => setCurrentSource('all')}
+                title={i18n.t(`SEARCH.SOURCES.${sourcesObj[currentSource]}.TEXT`)}
+                filterType={i18n.t('SEARCH.SOURCE')}
```

Comment:
strings should be refactored here too. Maybe I missed them.

Reply:
yeah, please go ahead and resolve this one @theamanjs

Reply:
hanji veerji, I already did resolve this one. Can you please review the PR https://github.com/KhalisFoundation/sttm-desktop/pull/1593